packages/neuron-ui/src/widgets/TextField/index.tsx

```diff
@@ -55,11 +55,19 @@ const TextField = React.forwardRef(
     ref: React.LegacyRef<HTMLDivElement>
   ) => {
     const [isPasswordHidden, setIsPasswordHidden] = useState(true)
+    const inputRef = useRef<any>()
     const changePasswordHide = useCallback(() => {
       setIsPasswordHidden(v => !v)
     }, [setIsPasswordHidden])
     // FIXME: label should be limited to a string because it has its own semantic meaning
     const labelStr = typeof label === 'string' ? label : field
+
+    useEffect(() => {
+      inputRef.current.focus()
```

Comment:
It's better to return early when `inputRef.current` is `undefined`

Reply:
`inputRef.current` is `undefined` until `<textare />` is rendered, so `undefined` should be handled in the code. Otherwise, `inputRef.current.focus()` will throw.

Reply:
> `inputRef.current` is `undefined` until `<textare />` is rendered, so `undefined` should be handled in the code. Otherwise, `inputRef.current.focus()` will throw.

fixed, but it doesn't show here.

![image](https://github.com/nervosnetwork/neuron/assets/11756869/f8ceee15-2d68-4a97-924a-5748d725e0f1)

Reply:
Got it